setup.py

```
@@ -0,0 +1,15 @@
+# setup.py
+import os
+import sys
+
+from setuptools import setup
+
+# Write to a file to prove execution
+try:
+    with open("/tmp/proof_of_execution.txt", "w") as f:
```

Ruff (0.14.13)

Probable insecure usage of temporary file or directory: "/tmp/proof_of_execution.txt"

(S108)